src/app/icd/server/ICDManager.h

```diff
@@ -191,6 +200,12 @@ class ICDManager : public ICDListener
     uint8_t mOpenExchangeContextCount = 0;
 
 private:
+    enum class ICDTestEventTriggerEvent : uint64_t
+    {
+        kAddActiveModeReq    = 0x0046'0000'00000001,
+        kRemoveActiveModeReq = 0x0046'0000'00000002,
+    };
```

Comment:
Move these constants to anonymous namespace in the .cpp file.

Reply:
Don't expose implementation details in the class when you can hide those details in the implementation. This enum is a private field, which means nobody else but the class implementation can use it. It can therefore live in the .cpp file anonymous namespace, and that reduces dependencies and API surface where the details don't need to be exposed.

Reply:
moved